src/android/io/sqlc/SQLitePlugin.java

```diff
@@ -66,6 +72,8 @@
     @Override
     public void initialize(CordovaInterface cordova, CordovaWebView webView) {
         super.initialize(cordova, webView);
+        logger = OSLogger.getInstance();
+        selfHealingEnabled = preferences.getBoolean("selfHealing", false);
```

Comment:
@bgracio we should have a more specific name for this preference since it's too generic. It should have more context like SQLCipher_Feature_Action (e.g.: SQLCipherSelfHealingEnabled) or Action_SQLCipher_Feature (e.g.: EnableSQLCipherSelfHealing). Your call :)

Reply:
done!